### ISSUE TITLE
Confused about (self.sparse_shape) in VoxelResBackBone8x and UNetV2

### ISSUE THREAD

**Issue by Ultra11nstinct** [open-mmlab/OpenPCDet#493]
class VoxelResBackBone8x(nn.Module):
    def __init__(self, model_cfg, input_channels, grid_size, **kwargs):
        super().__init__()
        self.model_cfg = model_cfg
        norm_fn = partial(nn.BatchNorm1d, eps=1e-3, momentum=0.01)
        `self.sparse_shape = grid_size[::-1] + [1, 0, 0]`

class UNetV2(nn.Module):
    def __init__(self, model_cfg, input_channels, grid_size, voxel_size, point_cloud_range, **kwargs):
        super().__init__()
        self.model_cfg = model_cfg
        `self.sparse_shape = grid_size[::-1] + [1, 0, 0]`

I wonder why the z axis of sparse_shape are always  1 more than grid_size?

**Comment by MengAaron** [comment]
#502 the same question
